### PR TITLE
Add support for Symfony 3.0

### PR DESCRIPTION
**Pull request by afurculita**

### CODE REVIEW

**Comment by Ener-Getick** [comment]
:+1: I added a travis build in #49 feel free to merge it if you want (I saw your PR after opening mine ...)

**Comment by afurculita** [comment]
@Ener-Getick great. I have just merged your changes into my PR. Thanks

File: composer.json [hunk 1]
```diff
@@ -21,8 +21,10 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.7",
-        "symfony/dependency-injection": "~2.7"
+        "symfony/framework-bundle": "~2.7|~3.0"
+    },
+    "require-dev": {
+        "symfony/phpunit-bridge": "~3.0"
```
Comment (Ener-Getick) [inline composer.json]: Just see that I made a mistake. Must be changed to ~2.7|~3.0

Comment (afurculita) [inline composer.json]: Just fixed that

**Comment by Ener-Getick** [comment]
:+1: everything works

**Comment by Ener-Getick** [comment]
ping @cedric-g @spike31

**Pull request merged**